disco_aws_automation/resource_helper.py

```diff
@@ -37,6 +37,15 @@ def tag2dict(tags):
     return {tag.get('Key'): tag.get('Value') for tag in tags or {}}
 
 
+def dict2tag(dicts):
+    """
+    Converts the list of key:value strings (example ["mykey:myValue", ...])
+    into a list of AWS tag dicts (example: [{'Key': 'mykey', 'Value': 'myValue'}, ...]
+    """
+    return [{'Key': tag_key_value[0], 'Value': tag_key_value[1]}
+            for tag_key_value in [key_value_option.split(":") for key_value_option in dicts]]
```

Comment:
`:` is an allowed value for tag keys and values in AWS. Would it be better to use another delimiter, or should we have some handling around the fact that a `key_value_option` might contain multiple `:` characters?

[AWS Tag Restrictions](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions)